### PR TITLE
Issue309

### PR DESCRIPTION
**Pull request by mattloose**
This pull request provides a new bulk file for promethION and will fix links in the documentation.

### CODE REVIEW

**Comment by Adoni5** [comment]
As we don't deploy the docs except on tags - Suggesting that we merge this and push the built docs to the `gh-pages` branch, rather than making a tag.

If that doesn't work (it should), we can maybe tag 2023.1.1.1, which is super fun.

File: README.md [hunk 1]
```diff
@@ -245,8 +245,10 @@ You _may_ need to add the host `127.0.0.1` in the MinKNOW UI.
 
 <details style="margin-top: 10px"><summary id="configuring-bulk-fast5-file"><h3 style="display: inline;">Configuring bulk FAST5 file Playback</h3></summary>
 
-Download an open access bulk FAST5 file, either [R9.4.1][bulk - R9] or [R10 (5khz)][bulk - R10 5khz].
+Download an open access bulk FAST5 file, either [R9.4.1][bulk - R9.4.1] or [R10 (5khz)][bulk - R10.4 5khz].
```
Comment (Adoni5) [inline README.md:248]:
```suggestion
Download an open access bulk FAST5 file, either [R9.4.1 4khz][bulk - R9.4.1] or [R10 (5khz)][bulk - R10.4 5khz].
```

**Comment by mattloose** [comment]
Sounds good.

It would be better to not have to do a release for docs.

**Review by Adoni5 — COMMENTED** [review]
Looks good to me! I've checked it all builds and renders correctly in the sphinx docs as well 👍🏼 
✅

File: README.md [hunk 2]
```diff
@@ -545,8 +549,9 @@ Nick Loman, Josh Quick, John Tyson, Jared Simpson, Ewan Birney, Alexander Senf,
 And for our Awesome Logo please checkout out [@tim_bassford](https://twitter.com/tim_bassford) from [@TurbineCreative](https://twitter.com/TurbineCreative)!
 
 <!-- end-epilog -->
-[bulk - R9]: https://s3.amazonaws.com/nanopore-human-wgs/bulkfile/PLSP57501_20170308_FNFAF14035_MN16458_sequencing_run_NOTT_Hum_wh1rs2_60428.fast5
-[bulk - R10 5khz]: https://s3.amazonaws.com/nanopore-human-wgs/bulkfile/GXB02001_20230509_1250_FAW79338_X3_sequencing_run_NA12878_B1_19382aa5_ef4362cd.fast5
+[bulk - R9.4.1]: https://s3.amazonaws.com/nanopore-human-wgs/bulkfile/PLSP57501_20170308_FNFAF14035_MN16458_sequencing_run_NOTT_Hum_wh1rs2_60428.fast5
+[bulk - R10.4 5khz]: https://s3.amazonaws.com/nanopore-human-wgs/bulkfile/GXB02001_20230509_1250_FAW79338_X3_sequencing_run_NA12878_B1_19382aa5_ef4362cd.fast5
+[bulk - promethION - R10.4 4khz]: https://s3.amazonaws.com/nanopore-human-wgs/bulkfile/PC24B243_20220512_1516_PAK21362_3H_sequencing_run_NA12878_sheared20kb_3d5147fc.fast5
```
Comment (Adoni5) [inline README.md:554]:
```suggestion

```
We don't actually need these at the bottom!

**Review by Adoni5 — DISMISSED** [review]
If we delete those bulk links at the bottom it LGTM

**Comment by mattloose** [comment]
LGTM

**Review by Adoni5 — APPROVED** [review]
Woah LGTM

**Pull request merged**